Multisig/UI/Assets/Ramper.swift

```diff
@@ -73,7 +99,7 @@ class Ramper: MoonpayCallbackInterface {
         )
 
         moonpay.doInit(config: config)
-        moonpay.show(mode: .WebViewOverlay())
+        moonpay.show(mode: .SafariOverlay())
```

Comment:
I guess, there is no way to make it work with WebViewOverlay? IIRC the integration was more seamless before.